main/index.js

```diff
@@ -1,9 +1,12 @@
 'use strict'
 
 const { app, dialog } = require('electron')
-const log = require('electron-log')
+const electronLog = require('electron-log')
 const path = require('node:path')
 
+console.log('Log file:', electronLog.transports.file.findLogPath())
```

Comment:
My intention here is to make it easier to locate the electron-log file when troubleshooting problems. I am using `console.log` instead of electron-log because it feels redundant to log the filename of the log file into that log file.

Let me know if you thing otherwise!